### DISCUSSION TITLE
How to disable routing at all?

### DISCUSSION THREAD

**Discussion by rpravm** [akvorado/akvorado#1010] [Q&A]
Is there a way to disable routing lookups? I don't need them, I have all ASN info in flows.
By default Akvorado logs billions of "routing: error while looking up" errors in inlet.

**Comment by vincentbernat** [comment]
As a workaround until a newer version, use `collect-asns: false`, `collect-as-paths: false`, and `collect-communities: false`.

However, the error message should be rate limited to 3 per minute. Isn't that the case?

**Reply by rpravm** [reply]
No, errors are not rate-limited. See this in logs on 1.9.0 instances. 

No problem, will wait for new versions! :) 
Thanks!
